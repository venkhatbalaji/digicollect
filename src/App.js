import * as React from "react";
import EmployeeDetails from './component/EmployeeDetails/EmployeeDetails';

function App() {
    return (
        <div className='page'>
            <EmployeeDetails />
        </div>
    )
}
export default App;