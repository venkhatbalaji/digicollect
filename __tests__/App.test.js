import React from 'react';
import App from '../src/App';
import { create } from 'react-test-renderer'

describe('App unit test',()=>{
    test('testing app button', () => {
    let tree = create(<App />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})

