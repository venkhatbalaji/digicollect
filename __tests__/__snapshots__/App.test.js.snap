// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App unit test testing app button 1`] = `
<div
  className="page"
>
  <div
    className="sc-furwcr jvxAQi"
  />
</div>
`;
